src/nvim/shada.c

```diff
@@ -10,9 +5,6 @@
 #include <stdint.h>
 #include <inttypes.h>
 #include <errno.h>
-#ifdef HAVE_UNISTD_H
-# include <unistd.h>
-#endif
```

Comment:
`ascii.h` including `os/*` stuff is not reliable. If correct unistd include is os_defs.h, it should be here explicitly.